lib/actions.ex

```diff
@@ -189,26 +375,52 @@ defmodule EctoShorts.Actions do
     end
   end
 
+  def delete(schema_data, opts) when is_list(schema_data) do
+    Enum.map(schema_data, &delete(&1, opts))
```

Comment:
We should resolve this to a singular result like we would normally do with `reduce_status_tuples`